pkg/polaris/polarisReportingMutator.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 Synopsys, Inc.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package polaris
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func GetPolarisReportingComponents(baseUrl string, polaris Polaris) (map[string]runtime.Object, error) {
+	content, err := GetBaseYaml(baseUrl, "polaris", polaris.Version, "reporting_base.yaml")
```

Comment:
Should this be: GetBaseYaml(baseUrl, `"reporting"`, polaris.Version, "reporting_base.yaml") ?

Reply:
@mphammer All the base yaml's are going to be under same folder. 

Ref : https://github.com/blackducksoftware/releases/tree/Development/polaris/2019.10.0.0.5116.0.0.757.0.0.1358